tests/Tests/100__deploy/110__auth_providers_and_rbac/110__rbac.robot

```diff
@@ -24,3 +33,23 @@ Verify User Can Spawn A Notebook
     Wait For RHODS Dashboard To Load
     Launch JupyterHub Spawner From Dashboard
     Spawn Notebook With Arguments    image=s2i-minimal-notebook    size=Default
+
+Verify User Is Unable To Spawn Notebook
+    [Documentation]    Verifies User is unable to access notebooks in jupyterhub
+    Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+    Login To RHODS Dashboard    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}
+    Wait For RHODS Dashboard To Load
+    Menu.Navigate To Page    Applications    Enabled
```

Comment:
not needed becase the when Dashboard is launched, "Enabled" is the first page shown

Reply:
Updated